scripts/linters/pylint_extensions_test.py

```diff
@@ -332,6 +332,29 @@ def test_hanging_indentation_with_a_comment_after_square_bracket(self):
         with self.checker_test_object.assertNoMessages():
             temp_file.close()
 
+    def test_hanging_indentation_with_a_if_statement_before(self):
+        node_with_no_error_message = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+        with utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                if 5 > 7:
+                self.post_json([
+                '(',
+                '', '', ''])""")
+        node_with_no_error_message.file = filename
+        node_with_no_error_message.path = filename
+
+        self.checker_test_object.checker.process_tokens(
+           pylint_utils.tokenize_module(node_with_no_error_message))
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
```

Comment:
Since the original bug was for an error message not being thrown, shouldn't we have a test to confirm that  an error is now thrown (instead of this test, which checks that an error is not thrown)?

Reply:
Done.